syntax = "proto3";

option java_multiple_files = true;
option java_package = "ru.itis.fact";

package fact;

service FactCalculator {
  rpc calculate_fact(Number) returns (stream Number) {}
}

message Number {
  int64 value = 1;
}

